packages/venia-concept/src/RootComponents/Product/Product.js

```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
 import { string, func } from 'prop-types';
 
+import { isUndefined } from 'util';
```

Comment:
No need for this guy, we can check if an object is `undefined` easily without pulling this in.

Reply:
Also, just an FYI that `isUndefined` is deprecated: https://nodejs.org/api/util.html#util_util_isundefined_object.